src/hotspot/os/windows/attachListener_windows.cpp

```diff
@@ -32,17 +32,24 @@
 #include <signal.h>             // SIGBREAK
 #include <stdio.h>
 
-// The AttachListener thread services a queue of operations. It blocks in the dequeue
-// function until an operation is enqueued. A client enqueues an operation by creating
+// The AttachListener thread services a queue of operation requests. It blocks in the dequeue
+// function until a request is enqueued. A client enqueues a request by creating
 // a thread in this process using the Win32 CreateRemoteThread function. That thread
 // executes a small stub generated by the client. The stub invokes the
-// JVM_EnqueueOperation function which checks the operation parameters and enqueues
-// the operation to the queue serviced by the attach listener. The thread created by
+// JVM_EnqueueOperation or JVM_EnqueueOperation_v2 function which checks the operation parameters
+// and enqueues the operation request to the queue serviced by the attach listener. The thread created by
```

Comment:
Nit: Just a suggestion to simplify this line of comment a little bit:
  // and enqueues the operation request for the attach listener. The thread created by

Reply:
I've made it "enqueues the operation request to the queue."